spec/rails_helper.rb

```diff
@@ -25,10 +26,12 @@
 # Checks for pending migration and applies them before tests are run.
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
do more research on including helper methods before I merge!

Reply:
We reviewed helpers and how they are useful when including certain methods globally throughout the app like authentication and how they are called through modules and placed usually in the /lib file.  They are different then models as models are restricted based on the class they are in based on my understanding!